### PR TITLE
made the SignUp and the LoginPage Ui again with a better design

### PR DESCRIPTION
**Pull request by SumitKumar-17**
Hi, This is with reference to Issue No.10, assigned to me as a part of GSSOC'24.
I have completed the frontend Design of the Sign Up and the Login page UI. it is also integrated with functionality and working.

### CODE REVIEW

File: frontend/src/app/login/page.tsx [hunk 1]
```diff
@@ -56,48 +56,94 @@ function LoginInner() {
...
-    </div>
+    </>
+
+
   );
 }
+
+
+
+
```
Comment (afeefuddin) [inline frontend/src/app/login/page.tsx:149]: Remove these unnecessary extra lines

**Comment by afeefuddin** [comment]
Deployment is failing with these errors 
```
134:16  Error: `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.  react/no-unescaped-entities
```

You need to fix that.

File: frontend/src/app/login/page.tsx (hunk 1 above)
Comment (SumitKumar-17) [reply frontend/src/app/login/page.tsx]: I have completed this .Please have a look on it.

**Comment by SumitKumar-17** [comment]
> Deployment is failing with these errors
> 
> ```
> 134:16  Error: `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.  react/no-unescaped-entities
> ```
> 
> You need to fix that.

Complted this error by using &pos.Please have a look on it .

**Comment by afeefuddin** [comment]
Squash all the commits into one.

**Pull request closed without merging**